### PR TITLE
Begin the transition to oclif with 'balena env add' (fix dropped leading zero in device UUID)

### PR DESCRIPTION
**Pull request by pdcastro**
Addresses issue #1150 by transitioning `balena env add` to oclif.

Depends-on: https://github.com/balena-io/balena-cli/pull/1197
Change-type: patch
Signed-off-by: Paulo Castro <paulo@balena.io>

### CODE REVIEW

**Comment by pdcastro** [comment]
Note that this PR was rebased on #1197 for the required Node 8 version bump -- please review the later commits only.

File: automation/capitanodoc/index.ts [hunk 1]
```diff
@@ -39,26 +39,41 @@ export async function renderMarkdown(): Promise<string> {
...
-
 		result.categories.push(category);
 	}
 
 	return markdown.render(result);
 }
 
+function importCapitanoCommands(jsFilename: string): CapitanoCommand[] {
+	// tslint:disable-next-line:no-var-requires
+	const actions: any = require(path.join(process.cwd(), jsFilename));
```
Comment (hedss) [inline automation/capitanodoc/index.ts]: Typings?

File: automation/capitanodoc/utils.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+// import { flags, flagUsages } from '@oclif/parser';
```
Comment (hedss) [inline automation/capitanodoc/utils.ts]: Accidentally left in?

File: automation/capitanodoc/index.ts (hunk 1 above)
Comment (pdcastro) [reply automation/capitanodoc/index.ts]: Done 👍

File: automation/capitanodoc/utils.ts (hunk 2 above)
Comment (pdcastro) [reply automation/capitanodoc/utils.ts]: Done 👍

**Pull request merged**